src/Referent/ManagedUsersFilter.php

```diff
@@ -23,14 +24,50 @@ class ManagedUsersFilter
     const PARAMETER_OFFSET = 'o';
     const PARAMETER_TOKEN = 't';
 
+    /**
+     * @Assert\Type("bool")
```

Comment:
You can add an empty private constructor and remove those type assertions, since the good type is return from the VO line 77 and following.